src/classes/UTIL_Describe.cls

```diff
@@ -787,12 +787,9 @@ public class UTIL_Describe {
 
         for (SObject rt : results) {
 
-            // Check RecordType IS available for the running user
             if (!Test.isRunningTest()) {
-                if (recordTypeInfos.get(rt.Id).isAvailable()) {
-                    recordTypeByDeveloperName.put(String.valueOf(rt.get('DeveloperName')), recordTypeInfos.get(rt.Id));
-                    recordTypeByName.put(String.valueOf(rt.get('Name')), recordTypeInfos.get(rt.Id));
-                }
+                recordTypeByDeveloperName.put(String.valueOf(rt.get('DeveloperName')), recordTypeInfos.get(rt.Id));
```

Comment:
@MatthewBlanski @skosgi1 it looks like we know longer need the if(!Test.isRunningTest()) check now because (unless I am misreading it) lines 791 and 792 are essentially the same thing that is in the } else { block.  I want to be sure as well that this fix is the right fix. Matt, I heard you mention something about doing this at a different level...lets be sure on this one as all eyes will be upon us.

Reply:
Looks like they are the same, updated it @lifewithryan